### PR TITLE
Test admin panel and react

### PR DESCRIPTION
**Pull request by koriUA**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,11 +23,36 @@
...
+    "bootstrap-css-only": "^4.1.1",
+    "classnames": "^2.2.5",
     "dotenv": "^6.2.0",
     "koa": "^2.0.0-alpha.8",
+    "koa-static": "^5.0.0",
     "koa2-router": "^1.0.6",
     "nodemailer": "^5.1.1",
-    "pm2": "^3.2.9"
+    "pm2": "^3.2.9",
+    "react": "^0.14.6",
```
Comment (vivitali) [inline package.json]: 0.14.6  -> https://reactjs.org/versions

Comment (vivitali) [reply package.json]: lets use latest version for all packages (e.g. react, react-redux, react-router)

File: package.json [hunk 2]
```diff
@@ -21,11 +23,36 @@
...
+    "react-dom": "^0.14.6",
+    "react-redux": "^5.0.2",
+    "react-router": "^3.0.2",
+    "redux": "^3.6.0",
+    "redux-logger": "^2.8.1",
+    "redux-promise-middleware": "^4.2.0",
+    "redux-saga": "^0.14.3",
+    "redux-thunk": "^2.2.0",
```
Comment (vivitali) [inline package.json]: I'm not sure we need both redux-saga": "^0.14.3",
    "redux-thunk": "^2.2.0", - I would prefer saga

**Review by vivitali — COMMENTED** [review]
`src/public/client.min.js` do we need this file?

File: package.json (hunk 2 above)
Comment (vivitali) [reply package.json]: current webpack version 4.29.6

**Review by rbelmega — APPROVED** [review]

**Review by vivitali — APPROVED** [review]

**Comment by koriUA** [comment]
> `src/public/client.min.js` do we need this file?

yes, currently we need it for prod. Maybe in the future we will remove it and will run build in prod.

**Pull request merged**